data/abilities/defense-evasion/e52e7918-ce55-4451-ac19-713a52487ae8.yml

```
@@ -0,0 +1,37 @@
+- tactic: defense-evasion
+  technique_name: BITS Jobs
+  technique_id: T1197
+  name: BITSAdmin Agent deployment
```

```suggestion
  name: Deploy Agent via BITSAdmin
```

just to avoid any confusion about a bitsadmin-based agent vs running an agent via bitsadmin